reco_utils/recommender/sar/sar_singlenode.py

```diff
@@ -17,6 +18,12 @@
 from reco_utils.recommender import sar
 
 
+SIMILARITY_TYPES = {
```

Comment:
This looks weird to me, if I understood what you want to do. Not a strong suggestion, but I've seen more often to have a list `SIMILARITY_TYPES=["cooccurrence", "jaccard",  "lift"]`. Then in `similarity_type=SIMILARITY_TYPES["JACCARD"]`, we could put directly "jaccard". Also, in the init we could throw an exception if similarity_type not in SIMILARITY_TYPES

Reply:
I will change it to an enum which will raise an error if an invalid type is provided